2021-11-09 16:22:40,927 INFO    MainThread:14472 [wandb_setup.py:_flush():69] setting env: {}
2021-11-09 16:22:40,927 INFO    MainThread:14472 [wandb_setup.py:_flush():69] setting login settings: {}
2021-11-09 16:22:40,927 INFO    MainThread:14472 [wandb_init.py:_log_setup():342] Logging user logs to D:\Users\Alex\Fichiers\Programmation\Recherche - Gabriel\scikit-learn\wandb\run-20211109_162240-1op56e8h\logs\debug.log
2021-11-09 16:22:40,927 INFO    MainThread:14472 [wandb_init.py:_log_setup():343] Logging internal logs to D:\Users\Alex\Fichiers\Programmation\Recherche - Gabriel\scikit-learn\wandb\run-20211109_162240-1op56e8h\logs\debug-internal.log
2021-11-09 16:22:40,928 INFO    MainThread:14472 [wandb_init.py:init():375] calling init triggers
2021-11-09 16:22:40,928 INFO    MainThread:14472 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-11-09 16:22:40,928 INFO    MainThread:14472 [wandb_init.py:init():424] starting backend
2021-11-09 16:22:40,929 INFO    MainThread:14472 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn, using: spawn
2021-11-09 16:22:40,933 INFO    MainThread:14472 [backend.py:ensure_launched():135] starting backend process...
2021-11-09 16:22:41,040 INFO    MainThread:14472 [backend.py:ensure_launched():139] started backend process with pid: 26092
2021-11-09 16:22:41,041 INFO    MainThread:14472 [wandb_init.py:init():429] backend started and connected
2021-11-09 16:22:41,042 INFO    MainThread:14472 [wandb_init.py:init():477] updated telemetry
2021-11-09 16:22:41,043 INFO    MainThread:14472 [wandb_init.py:init():500] communicating current version
2021-11-09 16:22:42,389 INFO    MainThread:14472 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.6 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-11-09 16:22:42,390 INFO    MainThread:14472 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-11-09 16:22:42,676 INFO    MainThread:14472 [wandb_init.py:init():540] starting run threads in backend
2021-11-09 16:22:47,399 INFO    MainThread:14472 [wandb_run.py:_console_start():1601] atexit reg
2021-11-09 16:22:47,400 INFO    MainThread:14472 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.WRAP
2021-11-09 16:22:47,400 INFO    MainThread:14472 [wandb_run.py:_redirect():1512] Wrapping output streams.
2021-11-09 16:22:47,401 INFO    MainThread:14472 [wandb_run.py:_redirect():1536] Redirects installed.
2021-11-09 16:22:47,402 INFO    MainThread:14472 [wandb_init.py:init():565] run started, returning control to user process
2021-11-09 16:22:47,402 INFO    MainThread:14472 [wandb_config.py:__setitem__():141] config set data = {'name': 'bike', 'n_jobs': 1} - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x00000270A8DB8280>>
2021-11-09 16:22:47,402 INFO    MainThread:14472 [wandb_run.py:_config_callback():843] config_cb data {'name': 'bike', 'n_jobs': 1} None
2021-11-09 16:23:56,174 INFO    MainThread:14472 [wandb_run.py:_atexit_cleanup():1571] got exitcode: 0
2021-11-09 16:23:56,174 INFO    MainThread:14472 [wandb_run.py:_restore():1543] restore
2021-11-09 16:23:56,817 INFO    MainThread:14472 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 968
  total_bytes: 968
}

2021-11-09 16:23:56,947 INFO    MainThread:14472 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 968
  total_bytes: 968
}

2021-11-09 16:23:57,111 INFO    MainThread:14472 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 968
  total_bytes: 968
}

2021-11-09 16:23:57,502 INFO    MainThread:14472 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 968
  total_bytes: 13697
}

2021-11-09 16:23:57,617 INFO    MainThread:14472 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 968
  total_bytes: 14029
}

2021-11-09 16:23:57,727 INFO    MainThread:14472 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14029
  total_bytes: 14029
}

2021-11-09 16:23:57,837 INFO    MainThread:14472 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14029
  total_bytes: 14029
}

2021-11-09 16:23:57,946 INFO    MainThread:14472 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14029
  total_bytes: 14029
}

2021-11-09 16:23:58,054 INFO    MainThread:14472 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14029
  total_bytes: 14029
}

2021-11-09 16:23:58,167 INFO    MainThread:14472 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14029
  total_bytes: 14029
}

2021-11-09 16:23:58,510 INFO    MainThread:14472 [wandb_run.py:_wait_for_finish():1693] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14029
  total_bytes: 14029
}

2021-11-09 16:23:58,625 INFO    MainThread:14472 [wandb_run.py:_wait_for_finish():1693] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 14029
  total_bytes: 14029
}

2021-11-09 16:23:59,785 INFO    MainThread:14472 [wandb_run.py:_show_summary():1848] rendering summary
2021-11-09 16:23:59,810 INFO    MainThread:14472 [wandb_run.py:_show_files():1915] logging synced files
